webapp/sass/layout/_sidebar-left.scss

```diff
@@ -39,12 +39,14 @@
         }
     }
 
-    .dropdown-menu {
+    .navbar-nav > li > .dropdown-menu {
         max-height: 80vh;
         max-width: 200px;
         overflow-x: hidden;
         overflow-y: auto;
         width: 200px;
+        margin-top: 3px;
+        margin-left: 2px;
```

Comment:
Sorry, that's what I get when re-editing changes directly on Github. :P 
The changes have been applied to the pull request